i18n/MonotoneMuseum/ko_kr.yml

```
@@ -25,65 +25,538 @@ ko_kr:
+          ＞ 대가 발생 없음
+      DTM:
+        name: 일그러짐표(저택・성)
+        items:
+        - |-
+          【올풀림】
+          세계는 금이 가고 올풀림이 출현한다. 올풀림에 닿은 자는 허무에 삼켜져 돌아올 수 없다.
+        - |-
+          【색채침식】
+          세계가 칠흑과 흐린 백색으로 칠해진다. 씬 내에 존재하는 생물과 무생물은 흑백의 모노톤으로 변한다. 이 효과는 일그러짐을 내리는 이형의 죽음으로 해제된다.
+        - |-
+          【미친 시계】
+          괘종시계나 시계탑이 미친듯이 돌아간다. 일그러짐을 받지 않는 경우 씬 내에 존재하는 모든 생물과 무생물이 2D6년의 시간이 지난 것 처럼 행동한다. 생물이라면 노화한다.
+        - |-
+          【흉월】
+          저택의 창문에서 흐린 백색의 달이 보인다. 불길한 달빛이 비추는 저택에 있는 모든 사람은 마음이 허무에 사로잡혀 스스로 죽음으로 향한다.
+        - |-
+          【무한도서관】
+          영원히 이어지는 도서관. 그 곳에 있는 책이 지식을 향한 유혹을 속삭인다. 【의지】 난이도 14의 판정에 실패한 경우 1D6시간 [낭패]를 받고, 같은 수치만큼 【MP】를 잃는다.
+        - |-
+          【흐린 백색의 방】
+          흐린 백색의 방에 갇혔다. 마음이 시들고 이형에 다가간다. 이 일그러짐이 1D6씬 (혹은 1D6시간) 이내에 사라지지 않는 경우 모든 캐릭터는 (PC를 포함해) 엑스트라가 되어 두번다시 자아를 되찾을 수 없다.
+        - |-
+          【악한 거울】
+          화장대나 손거울, 거울의 방 등에서 거울 속의 자신이 히죽히죽 이쪽을 보고있다. 모든 실잣는이가 하는 판정의 펌블치가 +3된다.
+        - |-
+          【일그러진 음색】
+          저택 내에 사람을 불안하게 만드는 음악이 울려퍼진다. 마음을 흔들어 자신 이외라면 누구나 괴물로 보이기 시작한다. 【재봉】난이도 8의 판정에 실패한 캐릭터는 【MP】가 즉시 0이 된다.
+        - |-
+          【암흑회랑】
+          빛조차 삼켜버리는 칠흑의 어둠이 차오르고 촛불이 모두 꺼진다. 어떤 마법적 빛을 PC가 가지고있지 않는 한 사격공격의 달성치는 -5된다. 무언가 암시능력이나 강력한 후각 등을 가졌다고 GM이 판단한 캐릭터는 이 효과를 받지 않는다.
+        - |-
+          【움직이는 식물】
+          저택의 식물, 덤불이나 장미 등이 움직여 공격온다. 【지각】 난이도 12의 판정에 실패하면 <베기> 5D6의 대미지를 입는다.
+        - |-
+          【검은 그림자】
+          일그러짐에 의해 저택(혹은 저택이 세워진 장소)에 살고있던 과거의 인물들이 그림자가 되어 되살아나 저택을 배회한다. 【지각】난이도 10의 판정에 실패한 PC는 발견되어 【MP】를 4D6점 잃는다.
+        - |-
+          【흉포한 짐승】
+          저택에서 기르고 있는 맹수나 저택에 자리잡은 생쥐, 새 등이 이형화해서 습격해온다. GM은 적절한 에너미를 등장시켜 전투를 해도 좋다.
+        - |-
+          【이를 드러내는 방】
+          방이 왜곡되어 들어온 이를 잡아먹기 위애 이빨을 드러낸다. 실내에 있는 모든 캐릭터는 2D6을 굴려, 나온 눈만큼 【HP】를 잃는다. 단 2가 나온 경우 즉시 【HP】가 0이 된다.
+        - |-
+          【작렬하는 난로】
+          난로에서 불꽃이 뿜어져나와 공격한다. 【지각】 난이도 12의 판정에 실패하면 【HP】를 4D6점 잃는다.
+        - |-
+          【무한회랑】
+          끝없이 이어지는 복도를 헤맨다. 어디까지고 똑같은 방이다. 1D6시간 (혹은 GM이 정한 시간)을 소비하고 같은 수치만큼  【HP】를 잃는다.
+        - |-
+          【기묘한 인형】
+          등장한 캐릭터와 같은 얼굴의 인형이나 꼭두각시가 나타나 말을 걸어온다. 환영인가 그렇게 만들어진 것인가는 일그러짐을 받아들일 때까지 알 수 없다. 등장한 PC는 씬 종료시에 【MP】를 2D6점 잃는다.
+        - |-
+          【움직이는 가구】
+          의자나 난로, 샹들리에 등의 가구 혹은 꼭두각시가 일그러짐의 영향으로 습격해온다. GM은 임의의 에너미를 등장시켜 전투를 해도 좋다.
+        - |-
+          【염상(炎上)】
+          저택이 불타오르기 시작한다. 이대로 머문다면 문답무용으로 죽을 것이다.
+      OT3:
+        name: 징조표ver3.0
+        type: 2D6
+        items:
+        - |-
+          【폭주충동】
+          마음 속 깊은 곳에서부터 충동이 치솟는다. 눈 앞의 생명을 찢어버리고싶다ㅡ당신의 의지와 반대로 그 팔이 휘둘러진다. 지근거리에 있는 캐릭터 전원에게 즉시 <재봉> 2D6점의 대미지를 준다. 전투 중이 아닐 경우 어디까지를 지근으로 할지는 GM이 판단할것. 대미지를 줄 대상이 없는 경우 이 연극 중 당신의 행동치를 +1한다.
+        - |-
+          【균열】
+          격통이 일고, 당신의 몸에 금이 간다. 당신은 [2D6+배우 레벨]점의 【HP】를 잃는다.
+        - |-
+          【칠흑의 날개】
+          당신의 등에 오한과 가려움이 일고, 칠흑의 날개가 피어난다. 이 연극 중, 당신은 마이너 액션으로 박탈치 1을 상승시켜 비행상태가 된다. (해제에도 마이너 액션 사용) 
```

The trailing space should be removed.
```suggestion
          당신의 등에 오한과 가려움이 일고, 칠흑의 날개가 피어난다. 이 연극 중, 당신은 마이너 액션으로 박탈치 1을 상승시켜 비행상태가 된다. (해제에도 마이너 액션 사용)
```